packages/table-react/documentation/Table.mdx

```diff
@@ -388,3 +389,15 @@ Hele raden blir automatisk klikkbar, samtidig som man skal sette inn en celle fo
     knobs={expandableTableExampleKnobs}
     codeExample={expandableTableExampleCode}
 />
+
+### Sticky tabell heading
+
+Tabellens header blir festet til toppen av skjermen og flyter over tabellinnholdet om man scroller lenger ned i tabellen enn skjermens høyde. For å sette en egen bakgrunnsfarge kan man overstyre CSS-variabelen `--jkl-table-head-sticky-color`.
+
+<DevExample
```

Comment:
I markdown er det `ComponentExample` som gjelder. `DevExample` er for de ulike devserverne rundtomkring.

Reply:
©🍝 :D